dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/preheat/DefaultPreheatService.java

```diff
@@ -766,7 +766,13 @@ private void collectScanTargets(Map<Class<?>, List<?>> targets) {
           }
 
           objects.forEach(
-              o -> list.addAll(ReflectionUtils.invokeMethod(o, property.getGetterMethod())));
+              o -> {
+                Collection<Object> propertyValue =
+                    ReflectionUtils.invokeMethod(o, property.getGetterMethod());
+                if (!org.apache.commons.collections4.CollectionUtils.isEmpty(propertyValue)) {
```

Comment:
we have our own `CollectionUtils.isNotEmpty` method that does the same thing.
Always better to use our own instead of the dep.